### ISSUE TITLE
Accessibility Error: Links must have discernible text

### ISSUE THREAD

**Issue by beckej13820** [omeka-s-modules/ItemCarouselBlock#10]
I have been doing automated accessibility testing on an Omeka S site using the Item Carousel Block. The accessibility tool I am using for the testing is Axe Dev Tools, a free browser extension created by Deque. https://www.deque.com/axe/devtools/

The error I am getting from the plugin is: Links must have discernible text.

![Screen Shot 2022-05-14 at 10 07 16 AM](https://user-images.githubusercontent.com/29982981/168429256-34e4328b-f5c4-4f46-bbaf-8004c69bc293.png)

The issue is the way that the plugin makes the entire block around the slider into a link, with no text associated directly with the link. 

This is a WCAG 2.1 level A violation, and Axe rates it as a "Serious" error.

There are multiple suggestions from Deque in how to fix this error:

> Fix at least one (1) of these issues:
> 
> - Element does not have text that is visible to screen readers
> - aria-label attribute does not exist or is empty
> - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
> - Element has no title attribute

I think it would be most appropriate is to use the link title attribute and to fill it with the item's name.

https://dequeuniversity.com/rules/axe/4.4/link-name?application=AxeChrome
